client/src/containers/ReviewForm.js

```diff
@@ -48,15 +50,53 @@ class ReviewForm extends Component {
     this.props.addToHistory(newHistory);
   }
 
+  listSymptoms(symptomList) {
+    return symptomList.map((singleSymptom, index) => {
+      return <li key={index}>{singleSymptom} </li>;
+    });
+  }
+
+  sendEmail() {
+    const { category, location, chosenSymptoms } = this.props;
+
+    const emailText = `Dear Mr. John Doe, I would like to report ${
+      category
+    } hazard in ${location}.
+        The inconveniences I am experiencing are: ${chosenSymptoms}.
+        Attached is a picture of the problem. Please take care of this issue right away.
+        Best regards,
+        A concerned citizen`;
+
+    const email = { email: emailText };
+    this.props.emailSending(email);
+    this.updateNext();
+  }
+
   render() {
     if (this.state.redirectHome) {
       return <Redirect to="/" />;
     }
     return (
       <div className="mw6 mw7-ns center ph3 ph3-ns">
         <div className="ph3">
-          <h2 className=" custom-font f3 tc">Review Report Page</h2>
-          <h3 className=" custom-font f5 tc">Under Construction</h3>
+
+          <div>
+            <p> Dear Mr. John Doe,</p>
```

Comment:
Instead of using a generic name, we do need to have the names of the appropriate authorities in the database. I suggest that for the MVP we choose certain locations that are associated to each one of us (1-2 location associated to Hoslack's email, 1-2 locations to mine, 1-2 locations to Marlen's) and depending on the location chosen we see the right name in the email template and the email is sent to the right person. Smething like this would truly show the functionality of the app.